rethinkdb.go

```diff
@@ -35,27 +27,24 @@ const (
 func init() {
 	// Set encoding package
 	encoding.IgnoreType(reflect.TypeOf(Term{}))
+}
 
-	Log = logrus.New()
-	Log.Out = ioutil.Discard // By default don't log anything
+// SetTags allows you to override the tags used when decoding or encoding
+// structs. The driver will check for the tags in the same order that they were
+// passed into this function. If no parameters are passed then the driver will
+// default to checking for the rethinkdb tag (the rethinkdb tag is always included)
+// Old-style gorethink tag is also supported but deprecated
+func SetTags(tags ...string) {
+	encoding.Tags = append(tags, encoding.TagName, encoding.OldTagName)
 }
 
 // SetVerbose allows the driver logging level to be set. If true is passed then
 // the log level is set to Debug otherwise it defaults to Info.
 func SetVerbose(verbose bool) {
 	if verbose {
-		Log.Level = logrus.DebugLevel
+		slog.SetLogLoggerLevel(slog.LevelDebug)
 		return
 	}
 
-	Log.Level = logrus.InfoLevel
-}
-
-// SetTags allows you to override the tags used when decoding or encoding
-// structs. The driver will check for the tags in the same order that they were
-// passed into this function. If no parameters are passed then the driver will
-// default to checking for the rethinkdb tag (the rethinkdb tag is always included)
-// Old-style gorethink tag is also supported but deprecated
-func SetTags(tags ...string) {
-	encoding.Tags = append(tags, encoding.TagName, encoding.OldTagName)
+	slog.SetLogLoggerLevel(slog.LevelInfo)
 }
```

Comment:
I agree that this should be removed later, but I don't think it should be removed as part of a PR. That PR is (mainly) about to replace Logrus. This function is probably used by some people out there. Even it is deprecated, the README still says:

> If you wish to use the `json` tags for RethinkDB-go then you can call `SetTags("rethinkdb", "json")` when starting your program, this will cause RethinkDB-go to check for `json` tags after checking for `rethinkdb` tags. By default this feature is disabled. This function will also let you support any other tags, the driver will check for tags in the same order as the parameters.

I would say remove this function and the related information + mention the removal in the README in a separate PR.

Reply:
OK